### PR TITLE
Fix bot not returning back after telepoting

### LINKED ISSUE #3037: dev snipe feature broken ?
bdcf2519d63555ac32246439fd17672896ee64e4
with 
#3014 (fix)
and its messing up the previous location 
before bdcf2519d63555ac32246439fd17672896ee64e4 it was working very nice but now when you run multiple bots and search  in the map additional to snipe its messing things really up

```
2016-08-08 11:51:39,398 [RecycleItems] [INFO] [item_discarded] Discarded 1x Potion (maximum 0).
2016-08-08 11:51:40,042 [RecycleItems] [INFO] [item_discarded] Discarded 2x Revive (maximum 0).
2016-08-08 11:51:40,579 [TransferPokemon] [INFO] [future_pokemon_release] Releasing Magikarp (CP 10/IV 0.44) based on rule: CP < 9 OR IV < 0.97
2016-08-08 11:51:41,302 [TransferPokemon] [INFO] [pokemon_release] Exchanged Magikarp [CP 10] [IV 0.44] for candy.
2016-08-08 11:51:52,028 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_teleport_to] Teleporting to Pikachu. (0.73km)
2016-08-08 11:51:52,028 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_encounter] Encountered Pokemon: Pikachu
2016-08-08 11:51:54,583 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_teleport_back] Teleporting back to previous location (40.7707074943, -73.9768963862)
2016-08-08 11:52:05,896 [PokemonCatchWorker] [INFO] [pokemon_appeared] A wild Pikachu appeared! [CP 193] [Potential 0.49] [S/A/D 1/13/8]
2016-08-08 11:52:08,793 [PokemonCatchWorker] [INFO] [vip_pokemon] This is a VIP pokemon. Catch!!!
2016-08-08 11:52:09,372 [PokemonCatchWorker] [INFO] [threw_berry] Threw a Razz Berry! Catch rate now: 30.03
2016-08-08 11:52:09,372 [PokemonCatchWorker] [INFO] [threw_pokeball] Used Ultraball, with chance 54.05 (49 left)
2016-08-08 11:52:10,404 [PokemonCatchWorker] [INFO] [pokemon_caught] Captured Pikachu! [CP 193] [Potential 0.49] [1/13/8] [+210 exp]
2016-08-08 11:52:15,906 [MoveToFort] [INFO] [moving_to_lured_fort] Moving towards pokestop Heckscher Ballfields - 0.06km (attraction of lure 0.13km)
2016-08-08 11:52:15,907 [MoveToFort] [INFO] [arrived_at_fort] Arrived at fort.
2016-08-08 11:52:17,648 [  SpinFort] [INFO] [spun_pokestop] Spun pokestop Heckscher Ballfields. Experience awarded: 50. Items awarded: {u'Pokeball': 2, u'Greatball': 1}
2016-08-08 11:52:20,653 [TransferPokemon] [INFO] [future_pokemon_release] Releasing Pikachu (CP 193/IV 0.49) based on rule: CP < 9 OR IV < 0.97
2016-08-08 11:52:21,306 [TransferPokemon] [INFO] [pokemon_release] Exchanged Pikachu [CP 193] [IV 0.49] for candy.
2016-08-08 11:52:29,412 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_teleport_to] Teleporting to Squirtle. (12947.95km)
2016-08-08 11:52:29,413 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_encounter] Encountered Pokemon: Squirrel
2016-08-08 11:52:32,048 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_teleport_back] Teleporting back to previous location (22.3365302665, 114.136735939)
2016-08-08 11:52:40,143 [PokemonCatchWorker] [INFO] [pokemon_appeared] A wild Squirtle appeared! [CP 479] [Potential 0.38] [S/A/D 2/14/1]
2016-08-08 11:52:43,994 [PokemonCatchWorker] [INFO] [vip_pokemon] This is a VIP pokemon. Catch!!!
2016-08-08 11:52:44,592 [PokemonCatchWorker] [WARNING] [softban] Failed to use berry. You may be soft banned
2016-08-08 11:52:44,592 [PokemonCatchWorker] [INFO] [threw_pokeball] Used Ultraball, with chance 24.99 (48 left)
2016-08-08 11:52:45,808 [PokemonCatchWorker] [INFO] [pokemon_vanished] Squirrel vanished!
2016-08-08 11:52:51,299 [MoveToFort] [INFO] [moving_to_fort] Moving towards pokestop Chirpy Chris  - 12946.65km
2016-08-08 11:52:58,057 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_updated_map] Updated PokemonGo-Map to 22.3366325493, 114.135584819
2016-08-08 11:53:02,498 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_teleport_to] Teleporting to Dratini. (2884.30km)
2016-08-08 11:53:02,498 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_encounter] Encountered Pokemon: Dratini
2016-08-08 11:53:05,115 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_teleport_back] Teleporting back to previous location (35.7132828916, 139.715192221)
2016-08-08 11:53:17,727 [PokemonCatchWorker] [INFO] [pokemon_appeared] A wild Dratini appeared! [CP 147] [Potential 0.53] [S/A/D 15/2/7]
2016-08-08 11:53:20,305 [PokemonCatchWorker] [INFO] [threw_pokeball] Used Pokeball, with chance 49.83 (124 left)
2016-08-08 11:53:21,303 [PokemonCatchWorker] [INFO] [pokemon_vanished] Dratini vanished!
2016-08-08 11:53:26,917 [MoveToFort] [INFO] [moving_to_fort] Moving towards pokestop 法鼓山道場 - 2883.31km
2016-08-08 11:53:50,589 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_teleport_to] Teleporting to Pikachu. (10841.11km)
2016-08-08 11:53:50,589 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_encounter] Encountered Pokemon: Pikachu
2016-08-08 11:53:53,144 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_teleport_back] Teleporting back to previous location (40.7693945909, -73.9763454099)
2016-08-08 11:54:05,427 [PokemonCatchWorker] [INFO] [pokemon_appeared] A wild Pikachu appeared! [CP 496] [Potential 0.64] [S/A/D 1/14/14]
2016-08-08 11:54:09,085 [PokemonCatchWorker] [INFO] [vip_pokemon] This is a VIP pokemon. Catch!!!
2016-08-08 11:54:09,639 [PokemonCatchWorker] [WARNING] [softban] Failed to use berry. You may be soft banned
2016-08-08 11:54:09,640 [PokemonCatchWorker] [INFO] [threw_pokeball] Used Ultraball, with chance 23.91 (47 left)
2016-08-08 11:54:10,677 [PokemonCatchWorker] [INFO] [pokemon_vanished] Pikachu vanished!
2016-08-08 11:54:16,196 [MoveToFort] [INFO] [moving_to_fort] Moving towards pokestop 林春雄墓碑 - 10839.83km
2016-08-08 11:54:28,813 [PokemonCatchWorker] [INFO] [pokemon_appeared] A wild Rattata appeared! [CP 159] [Potential 0.56] [S/A/D 12/8/5]
2016-08-08 11:54:31,142 [PokemonCatchWorker] [INFO] [threw_pokeball] Used Pokeball, with chance 45.14 (123 left)
2016-08-08 11:54:32,142 [PokemonCatchWorker] [INFO] [pokemon_vanished] Rattata vanished!
2016-08-08 11:54:38,123 [  SpinFort] [INFO] [pokestop_empty] Found nothing in pokestop Michelle Bernstein Landmark.
2016-08-08 11:54:38,124 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-08 11:54:38,124 [  SpinFort] [INFO] [softban] Probably got softban.
2016-08-08 11:54:49,836 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_teleport_to] Teleporting to Porygon. (16173.26km)
2016-08-08 11:54:49,836 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_encounter] Encountered Pokemon: Polygon
2016-08-08 11:54:52,406 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_teleport_back] Teleporting back to previous location (-6.22584795679, 106.797881332)
2016-08-08 11:55:02,804 [PokemonCatchWorker] [INFO] [pokemon_appeared] A 
```..

### PR DESCRIPTION
**Pull request by bixuanzju**
Short Description: 

Fixes:
- typo: last_long -> last_lon
- Fix #3037: bot not returning back after teleport

### CODE REVIEW

File: pokemongo_bot/cell_workers/move_to_map_pokemon.py [hunk 1]
```diff
@@ -335,7 +335,7 @@ def _teleport_back(self):
         self.emit_event(
             'move_to_map_pokemon_teleport_back',
             formatted=('Teleporting back to previous location ({last_lat}, '
-                       '{last_long})'),
+                       '{last_lon})'),
             data={'last_lat': last_position[0], 'last_lon': last_position[1]}
```
Comment (hackenho) [inline pokemongo_bot/cell_workers/move_to_map_pokemon.py]: You should also fix it here as well

**Comment by mhdasding** [comment]
duplicate of #3003

**Comment by fari-99** [comment]
after i see the log in #3037, it looks like the teleporting not in the same location after teleport.

**Comment by Philipp59** [comment]
@fari-99 not the problem of this fix this fix working the problem is located in bdcf2519d63555ac32246439fd17672896ee64e4

**Comment by bixuanzju** [comment]
@Philipp59 @fari-99 Since I am on it, let me fix it once and for all. Try it and tell me if this fixes the problem :)

**Comment by sffetlio** [comment]
@bixuanzju I've tested a little and it looks good.

**Pull request merged**